app/controllers/staff/application_controller.rb

```diff
@@ -26,6 +26,13 @@ def require_program_team
     end
   end
 
+  def require_reviewer
```

Comment:
So the require_staff check covers this already.  This is more explicit but the way permissions work for staff, reviewers are the least permissible staff members so a staff check allows in reviewers or higher.